plugins/ja/qml/keys/CursorKey.qml

```diff
@@ -51,6 +51,8 @@ ActionKey {
     }
 
     onPressAndHold: {
-        return;
+        fullScreenItem.cursorSwipe=true;
+	if(rightSide)
+		cursorSwipeArea.selectionMode=true;
```

Comment:
Please use space characters around `=` assignments, between `if` and open parenthesis, and `{}` around the code block inside the if statement. Also, 4 spaces for indentation, for consistency with the rest of the code here. Thanks.

Reply:
whoops my bad will work on that